### PR TITLE
zh-trans website/content/zh/docs/tasks/administer-cluster/guaranteed-…

### PR DESCRIPTION
**Pull request by zhangx501**
…scheduling-critical-addon-pods.md

根据英文版本删减，重新更新了中文译文，同时更改了部分译文，让阅读更流畅

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 1]
```diff
@@ -1,54 +1,46 @@
...
+{{% capture body %}}
 
-## 二次调度器：关键插件的调度保证
 
-二次调度器确保关键的插件总是能被调度（假定普通的 Pod 不存在时，集群具有足够的资源去运行关键插件 Pod）。
-如果调度器确定没有节点有足够资源去运行关键插件 Pod，假使有 Pod 已经运行在集群中（通过将关键插件 Pod 的条件 PodScheduled 设置为 false，原因设置为 Unschedulable  来指示），这时二次调度器通过清理一些 Pod 尽量去释放空间，然后调度器将调度该插件 Pod。
+<!--
+### Marking pod as critical
+-->
+### 将pod标记为	`critical`
```
Comment (Mr-Linus) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]: 空行太多

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 2]
```diff
@@ -1,54 +1,46 @@
...
 
+<!--
+To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
+->
+要将pod标记为	`critical` ，pod 必须在kube-system命名空间中运行（可通过标志配置），以及
+<!--
+* Have the priorityClassName set as "system-cluster-critical" or "system-node-critical", the latter being the highest for entire cluster. Alternatively, you could add an annotation `scheduler.alpha.kubernetes.io/critical-pod` as key and empty string as value to your pod, but this annotation is deprecated as of version 1.13 and will be removed in 1.14.
+-->
+* 将 `priorityClassName` 设置为 `system-cluster-critical` 或 `system-node-critical` ，后者是整个群集的最高级别。或者可以将注释 scheduler.alpha.kubernetes.io/critical-pod 作为密钥添加为空值，并将空字符串作为值添加到你的pod中，但此注释从1.13版本中开始不推荐使用，并将在1.14中删除。
```
Comment (Mr-Linus) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]:
```suggestion
* 将 `priorityClassName` 设置为 `system-cluster-critical` 或 `system-node-critical`，后者是整个群集的最高级别。或者可以将注释 `scheduler.alpha.kubernetes.io/critical-pod` 作为密钥添加为空值，并将空字符串作为值添加到你的 pod 中，但此注释从 1.13 版本中开始不推荐使用，并将在 1.14 中删除。
```

**Comment by markthink** [comment]
@zhangx501 文件有冲突，需要处理一下

**Comment by markthink** [comment]
@zhangx501 这个PR 时间有点久了且现在有冲突，麻烦处理一下，有问题可以找我。

**Comment by chenrui333** [comment]
@zhangx501 let us know if you need any help, thanks a lot for your contribution!!

**Comment by markthink** [comment]
@zhangx501 此 PR 有些时间了，需要解决一下冲突，有问题可以联系我。

**Comment by tengqm** [comment]
@zhangx501 ping

**Comment by tengqm** [comment]
stale for too long.

**Pull request closed without merging**